### PR TITLE
Update the README.md file to describe the repo.

### PR DESCRIPTION
**Pull request by pr3pony**

### CODE REVIEW

**Review by shadyof — COMMENTED** [review]
I thought the course designer just want us to change the first line? The description is not clear enough. XD

But don't worry, I think the main purpose is to let everyone have a basic idea about how to use github.
And seems you already have other project on github so I assume that you already know how to commit change or do code review, which is good!

**Comment by pr3pony** [comment]
I still can't merge this PR. It says "Review required: At least 1 approving review is required by reviewers with write access."

**Review by shadyof — APPROVED** [review]

**Comment by shadyof** [comment]
Oops, I think it's ready now?

**Pull request merged**
